src/specs/networking.md

```diff
@@ -66,3 +68,26 @@ Accepting a `WireTxPayForMessage` into the mempool requires different logic than
+| `position`    | `uint64`                                                                                    | The index of the row or column in the square.                                                                        |
+
+
+## StateFraudProof
+
+| name                       | type                                                                                     | description                                                                                                                                                                                                                                                                                  |
+|----------------------------|------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `headerForTxns`            | [Header](./data_structures.md#header)                                                    | Header containing `availableDataRoot`(./data_structures.md#header) of type [HashDigest](./data_structures.md#hashdigest) committing to transaction data.                                                                                              |
+| `headerForISRs`            | [Header](./data_structures.md#header)                                                    | Header containing `availableDataRoot`(./data_structures.md#header) of type [HashDigest](./data_structures.md#hashdigest) committing to intermediate state root data. `lastHeaderHash` within `headerForISRs` equals the hash of `headerForTxns`.|
+| `transactionShareProofs`   | [ShareProof](./data_structures.md#shareproof)`[]`                                        | ShareProof contains both the [Share](./data_structures.md#share) and [NamespaceMerkleTreeInclusionProof](./data_structures.md#namespacemerkletreeinclusionproof) for the Share. `isCol` of type `bool` is set to `false`.                                           |
+| `stateShareProofs`         | [ShareProof](./data_structures.md#shareproof)`[]`                                        | ShareProof contains both the [Share](./data_structures.md#share) and [NamespaceMerkleTreeInclusionProof](./data_structures.md#namespacemerkletreeinclusionproof) for the Share. `isCol` of type `bool` is set to `false`.                                           |
+| `wrapIndex`                | `uint64`                                                                                 | Index for connecting the [WrappedIntermediateStateRoot](./data_structures.md#wrappedintermediatestateroot) and [WrappedTransaction](./data_structures.md#wrappedtransaction) after shares are parsed.                                                                                      |
```

Comment:
I would just call this `index` or something.

Reply:
Edited.